### PR TITLE
Feat: total triggered and avg. resolution time graph

### PR DESCRIPTION
**Pull request by ahmadshaheer**
#### Related Issues / PR's
close https://github.com/orgs/SigNoz/projects/19/views/1?pane=issue&itemId=75226466

#### Screenshots

NA

### CODE REVIEW

File: frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx [hunk 1]
```diff
@@ -1,41 +1,113 @@
...
-			content={<PopoverContent />}
-			trigger="click"
-		>
-			{children}
-		</Popover>
+		<div className="alert-popover">
+			<Popover
+				showArrow={false}
+				placement="bottom"
+				color="linear-gradient(139deg, rgba(18, 19, 23, 1) 0%, rgba(18, 19, 23, 1) 98.68%)"
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx:70]: we can pass a class and set styles in the class

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx [hunk 2]
```diff
@@ -15,25 +17,34 @@ function ChangePercentage({
...
+		);
+	}
+	if (direction < 0) {
+		return (
+			<div className="change-percentage change-percentage--error">
+				<div className="change-percentage__icon">
+					<ArrowUpRight size={14} color="var(--bg-cherry-500)" />
+				</div>
+				<div className="change-percentage__label">
+					{percentage}% vs Last {duration}
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:39]: use % icon from lucide

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx [hunk 3]
```diff
@@ -88,13 +101,13 @@ function StatsCard({
...
 			</div>
 
-			{!isEmpty && (
-				<div className="stats-card__graph">
-					<div className="graph">
-						<DotChartOutlined className="graph-icon" />
-					</div>
+			<div className="stats-card__graph">
+				<div className="graph">
+					{!isEmpty && timeSeries.length > 1 && (
```
Comment (YounixM) [inline frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:106]: can timeSeries be undefined ?

File: frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts [hunk 4]
```diff
@@ -0,0 +1,670 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```
Comment (YounixM) [inline frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts:1]: Please make sure we are disabling eslint only in specific cases

File: frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts [hunk 5]
```diff
@@ -0,0 +1,670 @@
...
+function timelinePlugin(opts: any): any {
+	const { mode, count, fill, stroke, laneWidthOption, showGrid } = opts;
+
+	const pxRatio = devicePixelRatio;
+
+	const laneWidth = laneWidthOption ?? 0.9;
+
+	const laneDistr = SPACE_BETWEEN;
+
+	const font = `${round(14 * pxRatio)}px Arial`;
```
Comment (YounixM) [inline frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts:220]: Why arial font ?

**Review by YounixM — APPROVED** [review]

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx (hunk 2 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:39]: in the designs, we have text % (https://www.figma.com/design/jWPCZEqdeA0yDYvixLQ42k/Alerts?node-id=2819-7734&m=dev)

File: frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx [hunk 6]
```diff
@@ -1,41 +1,113 @@
...
+			)}
+			{!!relatedLogsLink && (
+				<Link
+					to={`${ROUTES.TRACES_EXPLORER}?${relatedLogsLink}`}
+					className="contributor-row-popover-buttons__button"
+				>
+					<div className="icon">
+						<DraftingCompass
+							size={14}
+							color={isDarkMode ? 'var(--bg-vanilla-400)' : 'var(--text-ink-400'}
```
[the remark this answers is not in the document]
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx:46]: Done

File: frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx (hunk 1 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/AlertPopover/AlertPopover.tsx:70]: I tried that, but it wasn't changing the background of popover.

File: frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx (hunk 3 above)
Comment (ahmadshaheer) [reply frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx:106]: I have added a default value of `[]` to `timeSeries` prop. even if it's undefined this won't break IMO.

File: frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts (hunk 4 above)
Comment (ahmadshaheer) [reply frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts:1]: Done

File: frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts (hunk 5 above)
Comment (ahmadshaheer) [reply frontend/src/lib/uPlotLib/plugins/timelinePlugin.ts:220]: Changed it to Geist Mono

**Pull request merged**